lib/ClangImporter/ClangImporter.cpp

```diff
@@ -2412,6 +2412,10 @@ Decl *ClangImporter::importDeclCached(const clang::NamedDecl *ClangDecl) {
   return Impl.importDeclCached(ClangDecl, Impl.CurrentVersion);
 }
 
+ArrayRef<ModuleDecl::ImportedModule> ClangImporter::getImportedModules() const {
+  return Impl.ImportedHeaderExports;
```

Comment:
If this is all it's going to do, I'd prefer to just ask for `getImportedHeaderModule()` from outside and not have this entry point.